### PR TITLE
add wan sum fuk command

### PR DESCRIPTION
**Pull request by mralext20**

### CODE REVIEW

**Comment by slice** [comment]
smh use recipes

**Comment by mralext20** [comment]
Added.

File: dog/ext/memes.py [hunk 1]
```diff
@@ -371,6 +371,19 @@ def open():
...
         avatar_im.close()
 
+    @commands.command()
+    @commands.cooldown(1, 5, commands.BucketType.user)
+    async def wansumfuk(self, ctx, image_source: converters.Image):
+        """ wan sum fuk? """
+        await Meme.recipe(ctx, {
+            'image': 'resources/memes/wansumfuk.png',
+            'render_as': 'wansumfuk.png',
+            'cache': [ (Image, (66, 66)) ],
```
Comment (slice) [inline dog/ext/memes.py]: should be `image_source`, not `Image`

File: dog/ext/memes.py [hunk 2]
```diff
@@ -371,6 +371,19 @@ def open():
...
+            'steps': [
+                { 'place': (Image, (60, 30)) },
```
Comment (slice) [inline dog/ext/memes.py]: ditto

File: dog/ext/memes.py [hunk 3]
```diff
@@ -371,6 +371,19 @@ def open():
         avatar_bio.close()
         avatar_im.close()
 
+    @commands.command()
```
Comment (slice) [inline dog/ext/memes.py]: `hidden=True`

**Comment by slice** [comment]
should work™

**Pull request merged**